modules/apps/portal-language/portal-language-override-test/src/testIntegration/java/com/liferay/portal/language/override/service/test/PLOEntryLocalServiceTest.java

```diff
@@ -89,6 +96,58 @@ public void testAddOrUpdatePLOEntry() throws Exception {
 				RandomTestUtil.randomString(), languageId, StringPool.BLANK));
 	}
 
+	@Test
+	public void testImportPLOEntries() throws IOException, PortalException {
+		String key1 = RandomTestUtil.randomString();
+		String key2 = RandomTestUtil.randomString();
+		String value1 = RandomTestUtil.randomString();
+		String value2 = RandomTestUtil.randomString();
+
+		String propertiesString = StringBundler.concat(
+			key1, StringPool.EQUAL, value1, StringPool.NEW_LINE, key2,
+			StringPool.EQUAL, value2);
+
+		_ploEntryLocalService.importPLOEntries(
+			TestPropsValues.getCompanyId(),
+			new ByteArrayInputStream(
+				propertiesString.getBytes(StandardCharsets.UTF_8)),
+			"en_US", TestPropsValues.getUserId());
+
+		Assert.assertEquals(value1, _language.get(LocaleUtil.US, key1));
+
+		Assert.assertEquals(value2, _language.get(LocaleUtil.US, key2));
+	}
+
+	@Test
+	public void testImportPLOEntriesRollback() throws IOException {
+		String key1 = "a-good-key";
+		String key2 = "empty-key";
+		String value1 = RandomTestUtil.randomString();
+
+		String propertiesString = StringBundler.concat(
+			key1, StringPool.EQUAL, value1, StringPool.NEW_LINE, key2,
+			StringPool.EQUAL);
+
+		try {
+			_ploEntryLocalService.importPLOEntries(
+				TestPropsValues.getCompanyId(),
+				new ByteArrayInputStream(
+					propertiesString.getBytes(StandardCharsets.UTF_8)),
+				"en_US", TestPropsValues.getUserId());
+
+			Assert.fail();
+		}
+		catch (PortalException portalException) {
+			Assert.assertTrue(
+				PLOEntryValueException.MustNotBeNull.class.isAssignableFrom(
+					portalException.getClass()));
+		}
```

Comment:
Is it possible to use the `_assertException` helper here? This seems like it's pretty much doing the same thing.

Reply:
Not anymore, we not have to check the exceptions